config/katello.migrations/210415175000-enable-katello-agent.rb

```diff
@@ -0,0 +1,3 @@
+if answers['foreman_proxy_content'] && answers['foreman_proxy_content']['enable_katello_agent'].nil?
```

Comment:
This only works if it's a hash. While unlikely, it could also be the boolean true.
```suggestion
if answers['foreman_proxy_content'].is_a?(Hash) && answers['foreman_proxy_content']['enable_katello_agent'].nil?
```